### ISSUE TITLE
MythTv Backend WEB Server - scrolling lists jump

### ISSUE THREAD

**Issue by jksj461** [MythTV/mythtv#942]
MythTV Version : v35-Pre-263-g5bdafa4228 
The 'Videos','Upcoming' and 'Recordings' lists have a dis concerting jump a few seconds after scrolling the list down.
The unscrolled lists are fine, the jump occurs only after scrolling down.
The effect is minor on a monitor but quite dis concerting on a tablet.
For example if I scroll to the bottom to view the last entry, the list is repositioned to the entry above making the last entry inaccessible.
The 'Recording Rules' page does not have the same issue.
I think this is a regresion in current master as I cannot remember seeing it before.

**Comment by bennettpeter** [comment]
Seems to be a bug in PrimeNG
https://github.com/orgs/primefaces/discussions/353
I recently upgraded to PrimeNG V16. It must be a new issue.

**Comment by bennettpeter** [comment]
Fixed in commits 6f9fdcb50c and fc1f8dd93b.
This is a PrimeNG workaround by forcing all rows to be the same height.
Removed themes that did not allow the height to be set.

**Comment by jksj461** [comment]
MythTV Version : v35-Pre-269-g6f9fdcb50c
Still doing it for me cannot access last line in 'Upcoming','Recordings'.
Presumably I have a bad build, if your sure its fixed. If you think so I will do a full build rather than incremental.

**Comment by jksj461** [comment]
ok issue is fixed in Google Chrome.
Still get the problem with Firefox.
However that is my default browser so the old version might be cached?

**Comment by bennettpeter** [comment]
I only tested with Chrome. They should be compatible. Check which theme is selected. Does it still jump around after scrolling?

Note there are two scroll bars on the page. Scroll them both down.

The two scroll bars are to allow you to scroll the top banner  off screen so that you can see the maximum number of entries on screen at one time without scrolling the column headings off screen.

**Comment by bennettpeter** [comment]
I tried with Firefox, and there is a problem. Firefox forces rows to be taller than chrome, thus will fit less rows on a page. If I build it with the bigger row size it works with both. I will would rather try to make it smarter so that it will work with any browser or theme and use the optimum row height..

**Comment by jksj461** [comment]
Works great in both browsers, many thanks.
